src/components/PostMedia.tsx

```diff
@@ -2,50 +2,40 @@ import React from "react";
 import ReactPlayer from "react-player";
 import { Carousel } from "antd";
 import { RightOutlined, LeftOutlined } from "@ant-design/icons";
-import {
-  ActivityContentImage,
-  ActivityContentLink,
-} from "@dsnp/sdk/core/activityContent";
+import { ActivityContentImage } from "@dsnp/sdk/core/activityContent";
 
 interface PostMediaProps {
   attachment: ActivityContentImage[] | undefined;
 }
 
 const PostMedia = ({ attachment }: PostMediaProps): JSX.Element => {
   const getPostMediaItems = () => {
-    return attachment?.map(
-      (
-        item: {
-          type: string;
-          mediaType: string;
-          url: string | ActivityContentLink;
-        },
-        index: number
-      ) => (
+    return attachment?.map((item, index) => {
+      return (
         <div key={index} className="PostMedia__cover">
           {item.type.toLowerCase() === "image" && (
             <a
-              href={item.url as string}
+              href={item.url[0].href}
```

Comment:
will the url always be saved as an array? that seems like overkill if each attachment in each item in the map has an array of length 1.  why not save it as a string?

Reply:
it's to do with the attachment actually being potentially different sizes or codecs.  See @rosalinekarr 's comment.  For now we're just going to always use the first one and not bother checking whether there are others there and if we should use them.